tooling/create-tauri-app/src/recipes/cljs.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+// import { join } from 'path'
+import { shell } from '../shell'
+import { Recipe } from '../types/recipe'
+
+const cljs: Recipe = {
+  descriptiveName: {
+    name: 'ClojureScript (https://clojurescript.org/)',
+    value: 'cljs'
+  },
+  shortName: 'cljs',
+  extraNpmDevDependencies: [],
+  extraNpmDependencies: [],
+  configUpdate: ({ cfg }) => ({
+    ...cfg,
+    distDir: `../public`,
+    devPath: 'http://localhost:3000',
+  }),
+  preInit: async ({ cwd, cfg, ci, packageManager }) => {
+    // create-cljs-app creates the folder for you
+    if (packageManager === 'yarn') {
+      await shell(
+        'yarn',
+        [
+          'create',
+          'cljs-app',
+          `${cfg.appName}`
+        ],
+        {
+          cwd
+        }
+      )
+    } else {
+      await shell(
+        'npx',
+        [
+          'create-cljs-app@latest',
+          `${cfg.appName}`,
+        ],
+        {
+          cwd
+        }
+      )
+    }
+  },
+  postInit: async ({ cfg, packageManager }) => {
+    console.log(`
+    Your installation completed.
+
+    $ cd ${cfg.appName}
+
+    Run Tauri and Shadow CLJS in separate terminals:
+
+    $ ${packageManager === 'npm' ? 'npm run' : packageManager} start
```

Comment:
Why though ? The whole purpose of `create-tauri-app` is people won't have to do this because we pre-configured `beforeDevCommand` and `beforeBuildCommand` for them.

Reply:
Seems reasonable to me. I just don't know how to overwrite the scripts from `create-cljs-app` as it has `start` and `build` already.

Reply:
you don't need to overwrite them, you need to specify them in `beforeDevCommand` and `beforeBuildCommand` (see https://github.com/tauri-apps/tauri/pull/2843#discussion_r744153972) and `tauri dev` or `tauri build` will run them automatically.

Reply:
Ah ok, I didn't understand that's what `before(Dev|Build)Command` was doing.